.travis.yml

```diff
@@ -13,13 +16,21 @@ addons:
       - google-chrome-stable
 install:
   - sudo pip install -r requirements.txt
-  - npm install gatsby -g
+  - npm install -g gatsby
+  - npm install -g babel-cli
+  - npm install -g cross-env
 before_script:
   - wget "http://chromedriver.storage.googleapis.com/2.33/chromedriver_linux64.zip"
   - unzip chromedriver_linux64.zip
   - sudo mv chromedriver /usr/local/bin
+  - docker run -d -p 8080:8080 plone/plone.restapi
```

Comment:
@sneridagh I'd prefer to get away without docker here. We can build the plone instance on travis and then let robot framework fire up the instance.